src/unifrac_task.hpp

```diff
@@ -402,37 +402,40 @@ namespace SUCMP_NM {
     template<class TFloat>
     class UnifracUnweightedTask : public UnifracTask<TFloat,uint64_t> {
       public:
-        // use the smaller size for historical reasons
-        static const unsigned int step_size = 16*4/sizeof(TFloat);
+#ifdef _OPENACC
+        static const unsigned int step_size = 64*4/sizeof(TFloat);
+#else
+        // CPUS like a smaller step_size
+        static const unsigned int step_size = 64*4/sizeof(TFloat);
```

Comment:
Isn't this the same as the `ifdef _OPENACC` line?

Reply:
Argh... last minute typo.
Let me fix and re-benchmark.

Reply:
Removed ifdef